chat-plugins/lottery.js

```diff
@@ -0,0 +1,114 @@
+/**
+ *
+ * Lottery.js Made By Dragotic.
+ * A Lottery plugin that allows you to draw lottery and gain great rewards.
+ */
+
+'use strict';
+
+const color = require('../config/color');
+
+// Stores all the lottery tickets
+let allLotteryTickets = [];
```

Comment:
wouldn't having a single tickets object work the best? something like : 
`{
    'user1' : '[ticket 1, ticket 2]',
}`